resources.html

```diff
@@ -24,6 +24,8 @@
 
 <ul>
 
+<li>All User Guides for <a href="http://downloads.openmicroscopy.org/help/archives/OMERO_User_Guides_5_3_0.zip">Version 5.3.0</a> (Zipped PDFs 159 MB)</li>
```

Comment:
let's remove them in another PR that's a better idea.
I will remove all of the links, if people request we can also point them to the archives